main/Checkbox/index.tsx

```diff
@@ -63,11 +72,10 @@ interface CheckboxProps {
   boxSize?: number;
   boxColor?: string;
   defaultChecked?: boolean;
-  disabled?: boolean;
   onPress?: (value: string | number) => void;
   labelSize?: number;
   labelColor?: string;
-  item: { label: string; value: string | number; selected: boolean; }
+  item: { label: string; value: string | number; selected: boolean; disabled?: boolean;}
```

Comment:
I would be great to make the new interface for the checkbox data type. 
for example

```
interface CheckboxItem {
  label: string,
 ...
}
```

And you will replace this line as below. 

```
item: CheckboxItem;
```

And it can be used here as well.

https://github.com/dooboo-ui-turtle/dooboo-ui/blob/a11d9380cd879472d56a3176218984d1058d3d37/main/Checkbox/index.tsx#L64